src/api/db/structure.sql

```diff
@@ -365,7 +366,7 @@ CREATE TABLE `configurations` (
   `change_password` tinyint(1) DEFAULT '1',
   `hide_private_options` tinyint(1) DEFAULT '0',
   `gravatar` tinyint(1) DEFAULT '1',
-  `enforce_project_keys` tinyint(1) DEFAULT '0',
+  `enforce_project_keys` tinyint(1) DEFAULT '1',
```

Comment:
@adrianschroeter @hennevogel do we want enforce_project_keys enabled or disabled by default?